### PR TITLE
igraph and py-igraph: fix for older macOS

### PR DESCRIPTION
**Pull request by szhorvat**
#### Description

Fixes compilation on older macOS, see error here: https://build.macports.org/builders/ports-10.10_x86_64-builder/builds/194208/steps/install-port/logs/stdio

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.14.6 18G9323 x86_64
Xcode 11.3.1 11C504

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by szhorvat** [comment]
@nerdling I bumped the revision, but there is no functional change here. Should I keep the revision on `0` in that case?

**Comment by szhorvat** [comment]
I reverted the revision to `0` as this change would make no difference on systems where compilation did not fail.

**Review by nerdling — APPROVED** [review]

File: math/igraph/Portfile [hunk 1]
```diff
@@ -52,7 +52,8 @@ configure.args-append   -DUSE_CCACHE=OFF \
                         -DIGRAPH_USE_INTERNAL_GMP=OFF \
                         -DIGRAPH_USE_INTERNAL_LAPACK=OFF \
                         -DIGRAPH_USE_INTERNAL_PLFIT=ON \
-                        -DIGRAPH_OPENMP_SUPPORT=OFF
+                        -DIGRAPH_OPENMP_SUPPORT=OFF \
+                        -DIGRAPH_WARNINGS_AS_ERRORS=OFF # temporary fix, remove for 0.10.1
```
Comment (reneeotten) [inline math/igraph/Portfile:56]: are you sure you can add these type of comments in `configure.args-append` - I surely doubt it....

**Review by reneeotten — CHANGES_REQUESTED** [review]
the CI fails so that should be fixed - are you sure these comments are allowed/correct...?

File: math/igraph/Portfile (hunk 1 above)
Comment (szhorvat) [reply math/igraph/Portfile]: Thank you for pointing out the problem. This is fixed now.

**Review by reneeotten — APPROVED** [review]

**Pull request merged**